caput/scripts/runner.py

```diff
@@ -308,6 +315,16 @@ def queue(configfile, submit=False, lint=True):
     if sfile != dfile:
         shutil.copyfile(sfile, dfile)
 
+    if "modules" in rconf and rconf["modules"]:
+        modules = rconf["modules"]
+        modules = (modules,) if isinstance(modules, str) else modules
+        modstr = "module purge\nmodule load "
+        modstr += "\nmodule load ".join(modules)
```

Comment:
Note that if the module does not exist, the pipeline job will continue to try to run. `module load` does not return an error code.